### PR TITLE
Add alert feedback to Profile Completion Modules

### PR DESCRIPTION
**Pull request by lorenjohnson**

### CODE REVIEW

File: src/components/ProfileBioModule.js [hunk 1]
```diff
@@ -41,7 +41,10 @@ export default class ProfileBioModule extends Component {
     const { value } = this.state
     const { dispatch } = this.context
     trackEvent(ADDED_BIO, {context: 'onboarding'})
-    return dispatch(updateCurrentUser({bio: value}))
+    return Promise.all([
+      dispatch(updateCurrentUser({bio: value})),
+      dispatch(notify('Thanks for sharing!', {type: 'info'}))
```
Comment (Connoropolous) [inline src/components/ProfileBioModule.js]: all looks good code wise, the suggestion I have here is related to wording. I'd avoid the use of the word 'thanks' which installs the mental model that they just did something for hylo, or for us, when really we want them to know what they just did will benefit them. I don't think we want to come across as data collectors, and its all in the subtleties. I would suggest something simpler for the time being, just affirming, like: "Bio updated successfully." while we still have questions about the 'personality' of hylo.

File: src/components/ProfileSkillsModule.js [hunk 2]
```diff
@@ -30,7 +30,10 @@ export default class ProfileSkillsModule extends Component {
   save = () => {
     const { tags } = this.state
     const { dispatch } = this.context
-    return dispatch(updateCurrentUser({tags}))
+    return Promise.all([
+      dispatch(updateCurrentUser({tags})),
+      dispatch(notify('You\'ve got skills!', {type: 'info'}))
```
Comment (Connoropolous) [inline src/components/ProfileSkillsModule.js]: Same thing here for now, I would recommend 'Skills added successfully.'

Comment (lorenjohnson) [reply src/components/ProfileSkillsModule.js]: 👍

File: src/components/ProfileBioModule.js (hunk 1 above)
Comment (lorenjohnson) [reply src/components/ProfileBioModule.js]: 👍

**Review by Connoropolous — APPROVED** [review]

**Pull request merged**